system-int-test/test/cmr/system_int_test/search/autocomplete/suggestion_reindex_test.clj

```diff
@@ -14,25 +15,21 @@
    [cmr.system-int-test.utils.ingest-util :as ingest]
    [cmr.system-int-test.utils.search-util :as search]))
 
-(use-fixtures :each (join-fixtures
-                      [(ingest/reset-fixture {"provguid1" "PROV1"})
-                       hu/grant-all-humanizers-fixture
-                       hu/save-sample-humanizers-fixture]))
 
 (defn compare-autocomplete-results
- "Compare expected to actual response for the following:
- - Items are ordered by score in descending order
- - Ensure that the other fields match"
- [actual expected]
- (let [scores (map :score actual)
-       actual-scores-descending? (->> actual
-                                      (map :score)
-                                      (apply >=))
-       ; we don't need to actually compare scores
-       expected (map #(dissoc % :score) expected)
-       actual (map #(dissoc % :score) actual)]
-   (is (true? actual-scores-descending?))
-   (is (= expected actual))))
+  "Compare expected to actual response for the following:
+  - Items are ordered by score in descending order
+  - Ensure that the other fields match"
+  [actual expected]
+  (let [scores (map :score actual)
+        actual-scores-descending? (->> actual
+                                       (map :score)
+                                       (apply >=))
+                                        ; we don't need to actually compare scores
```

Comment:
nit: Change indentation of this comment to be over what it is referring to. Use two `;;` for comments on their own line.